issue with npm install

Hello,
I keep facing the issue with the below error with `npm install` on my local after I run `npm install @upstash/redis` although this error is being bypassed with `yarn install`
```
node_modules/@upstash/redis/zmscore-uDFFyCiZ.d.ts:283:5 - error TS5084: Tuple members must all have names or all not have names.

283     ...TRest
```

Since my code is already set up with `npm` (no issue without this new dependency), it is hard to change everything with `yarn` only for this dependency.
Do you have any clue how to make it work with `npm`?

Thanks!

--- a/pkg/commands/bitfield.ts
+++ b/pkg/commands/bitfield.ts
@@ -4,7 +4,7 @@ import { Command, type CommandOptions } from "./command";
 type SubCommandArgs<TRest extends unknown[] = []> = [
   encoding: string, // u1 - u63 | i1 - i64
   offset: number | string, // <int> | #<int>
-  ...TRest,
+  ...rest: TRest,
 ];
 
 /**

